### PR TITLE
Administrator View UI Updates

### PR DESCRIPTION
**Pull request by newlinewebdesign**
BS5 / Core Joomla html. language file updates

### CODE REVIEW

File: language/backend/en-GB/en-GB.com_j2store.ini [hunk 1]
```diff
@@ -118,16 +118,20 @@ J2STORE_ADVANCED_SETTINGS="Advanced Settings"
...
+
 J2STORE_POSTCONFIG_STORE_SETUP_DONE_ALREADY="First run wizard was already executed. If you need to alter the settings, please go to J2Store -> Configuration"
 
 J2STORE_DEFAULT_TAX_RATE="Default Tax Rate (optional)"
 J2STORE_DEFAULT_TAX_RATE_DESC="You can choose to create a default tax rate. You can also do this after completing this wizard. This is purely optional"
-J2STORE_POSTCONFIG_LBL_MANDATORYINFO="Mandatory Information"
+J2STORE_POSTCONFIG_LBL_MANDATORYINFO="Free Version License / Support"
 J2STORE_AGREE_SET_UP_STORE="Agree & Setup Store"
-J2STORE_POSTCONFIG_LBL_ACCEPTLICENSE="I have read, understood and accept the license of the software"
+J2STORE_POSTCONFIG_LBL_ACCEPTLICENSE="I have read, understand, and accept the license of the software."
```
Comment (brianteeman) [inline language/backend/en-GB/en-GB.com_j2store.ini:134]: this should be reverted - it was correct english

File: administrator/components/com_j2store/views/apps/tmpl/default.php [hunk 2]
```diff
@@ -186,7 +186,7 @@
 				<?php echo $this->pagination->getListFooter();?>
             </nav>
             <div class="text-center mt-2 mb-4 px-4">
-                <i class="fa-4x mb-2 fa-solid fa-cart-plus"></i>
+                <i class="fa-4x mb-2 fa-solid fas fa-cart-plus"></i>
```
Comment (brianteeman) [inline administrator/components/com_j2store/views/apps/tmpl/default.php:189]: this should really be a span and not i
i can do a separate PR as this is across the entire repo if you want

File: language/backend/en-GB/en-GB.com_j2store.ini (hunk 1 above)
Comment (newlinewebdesign) [reply language/backend/en-GB/en-GB.com_j2store.ini:134]: @brianteeman the verbs read, understand, and accept should all be in the present (or past) tense to maintain parallel structure. the previous language string is using a mix of both past and present tense. I don't care either which way we go, but we should be consistent.
The other alternative could be  I have read, understood, and accepted the license of the software?

File: administrator/components/com_j2store/views/apps/tmpl/default.php (hunk 2 above)
Comment (newlinewebdesign) [reply administrator/components/com_j2store/views/apps/tmpl/default.php:189]: if you can clean up the whole repo for those items, that works for me. I am still getting used to the Joomla method so bear with me...

File: language/backend/en-GB/en-GB.com_j2store.ini (hunk 1 above)
Comment (brianteeman) [reply language/backend/en-GB/en-GB.com_j2store.ini:134]: i have **read** the license, I have **understood** the licence and by clicking here I **accept** the licence

Comment (newlinewebdesign) [reply language/backend/en-GB/en-GB.com_j2store.ini:134]: even better.

**Pull request merged**